.env.example

```diff
@@ -0,0 +1,2 @@
+REACT_APP_PRODUCTION_API=<Production Url>
+REACT_APP_STAGING_API=<Staging Url>
```

Comment:
We then delete the separate ENV variable for staging.

Reply:
The production and development urls are different and are both on netlify